src/platform/silabs/SiWx917/wifi/wfx_host_events.h

```diff
@@ -57,77 +57,77 @@
 #define CONVERT_MS_TO_SEC(TimeInMS) (TimeInMS / 1000)
 
 // WLAN related Macros
-#define ETH_FRAME 0
-#define CMP_SUCCESS 0
-#define BSSID_MAX_STR_LEN 6
-#define MAC_ADDRESS_FIRST_OCTET 6
-#define AP_START_SUCCESS 0
-#define BITS_TO_WAIT 0
-#define CONNECTION_STATUS_SUCCESS 1
-#define IP_STATUS_FAIL 0
-#define GET_IPV6_SUCCESS 1
-#define GET_IPV6_FAIL 0
-#define BEACON_1 0
-#define CHANNEL_LIST (const uint8_t *) 0
-#define CHANNEL_COUNT 0
-#define IE_DATA (const uint8_t *) 0
-#define IE_DATA_LENGTH 0
-#define BSSID_SCAN (const uint8_t *) 0
-#define CHANNEL_0 0
-#define PREVENT_ROAMING 1
-#define DISABLE_PMF_MODE 0
-#define STA_IP_FAIL 0
-#define IP_STATUS_SUCCESS 1
-#define ACTIVE_CHANNEL_TIME_100 100
-#define PASSIVE_CHANNEL_TIME_0 0
-#define PROBE_NUM_REQ_1 1
-
-#define PINOUT_CLEAR_STATUS 0
-#define TICKS_TO_WAIT_0 0
-#define TICKS_TO_WAIT_3 3
-#define TICKS_TO_WAIT_1 1
-#define TICKS_TO_WAIT_500 500
+#define ETH_FRAME (0)
+#define CMP_SUCCESS (0)
+#define BSSID_MAX_STR_LEN (6)
+#define MAC_ADDRESS_FIRST_OCTET (6)
+#define AP_START_SUCCESS (0)
+#define BITS_TO_WAIT (0)
+#define CONNECTION_STATUS_SUCCESS (1)
+#define IP_STATUS_FAIL (0)
+#define GET_IPV6_SUCCESS (1)
+#define GET_IPV6_FAIL (0)
+#define BEACON_1 (0)
+#define CHANNEL_LIST ((const uint8_t *) 0)
+#define CHANNEL_COUNT (0)
+#define IE_DATA ((const uint8_t *) 0)
+#define IE_DATA_LENGTH (0)
+#define BSSID_SCAN ((const uint8_t *) 0)
+#define CHANNEL_0 (0)
+#define PREVENT_ROAMING (1)
+#define DISABLE_PMF_MODE (0)
+#define STA_IP_FAIL (0)
+#define IP_STATUS_SUCCESS (1)
+#define ACTIVE_CHANNEL_TIME_100 (100)
+#define PASSIVE_CHANNEL_TIME_0 (0)
+#define PROBE_NUM_REQ_1 (1)
+
+#define PINOUT_CLEAR_STATUS (0)
+#define TICKS_TO_WAIT_0 (0)
+#define TICKS_TO_WAIT_3 (3)
+#define TICKS_TO_WAIT_1 (1)
+#define TICKS_TO_WAIT_500 (500)
 
 // TASK and Interrupt Macros
-#define SUCCESS_STATUS 1
-#define LINK_UP 1
-#define LINK_DOWN 0
-#define MAC_48_BIT_SET 1
-#define STRUCT_PBUF (struct pbuf *) 0
-#define PRIORITY_0 0
-#define HEX_VALUE_FF 0XFF
+#define SUCCESS_STATUS (1)
+#define LINK_UP (1)
+#define LINK_DOWN (0)
+#define MAC_48_BIT_SET (1)
+#define STRUCT_PBUF ((struct pbuf *) 0)
+#define PRIORITY_0 (0)
+#define HEX_VALUE_FF (0XFF)
 
 // Timer Delay
-#define MAX_XLEN 16
-#define MIN_XLEN 0
-#define PINOUT_CLEAR 0
-#define PINOUT_SET 1
-#define WFX_SPI_NVIC_PRIORITY 5
-#define WFX_GPIO_NVIC_PRIORITY 5
-#define CB_VALUE (DMADRV_Callback_t) 0
+#define MAX_XLEN (16)
+#define MIN_XLEN (0)
+#define PINOUT_CLEAR (0)
+#define PINOUT_SET (1)
+#define WFX_SPI_NVIC_PRIORITY (5)
+#define WFX_GPIO_NVIC_PRIORITY (5)
+#define CB_VALUE ((DMADRV_Callback_t) 0)
 
 /* TIMER_TICKS_TO_WAIT Specifies the time, in ticks, that the calling task should
  * be held in the Blocked state to wait for the start command to be successfully
  * sent to the timer command queue.
  */
-#define TIMER_TICKS_TO_WAIT_0 pdMS_TO_TICKS(0)
+#define TIMER_TICKS_TO_WAIT_0 (pdMS_TO_TICKS(0))
```

Comment:
not sure we need a define for 0

Reply:
The define doesn't bother me but i wonder why a wait of 0 is needed

Reply:
Yes it is odd , and I checked it , we are not using it in our code.
So removed this MACRO